src/haconiwa/scan/cli.py

```
@@ -0,0 +1,330 @@
+    
+    models = scanner.list_all_models(
+        category=category,
+        provider=provider
+    )
+    
+    formatter = OutputFormatter()
+    output = formatter.format_model_list(models, output_format)
+    typer.echo(output)
+
+@scan_app.command("generate-parallel-config")
+def generate_parallel_config(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)